tutorials/nlp/Multitask_Prompt_and_PTuning.ipynb

```diff
@@ -0,0 +1,1184 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed04f167",
+   "metadata": {},
+   "source": [
+    "The `MegatronGPTPromptLearningModel` class expects datasets to be a list of `.json` or `.jsonl` file paths. You can give it multiple datasets at once like we did above.  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4e021b24",
+   "metadata": {},
+   "source": [
+    "### Prompt Formatting\n",
+    "Now that we have our datasets set, lets define what we want the prompt for each task to look like. \n",
+    "\n",
+    "To customize different prompts for different tasks, we simply need to specify the prompt task template in the config file. The virtual token markers `<|VIRTUAL_PROMPT_#|>` signify where you want virtual tokens to be placed in the template string. `<|VIRTUAL_PROMPT_0|>`, `<|VIRTUAL_PROMPT_1|>`, and `<|VIRTUAL_PROMPT_2|>` indicate where a number of virtual tokens matching the values given at `virtual_token_splits[0]`, `virtual_token_splits[1]` and `virtual_token_splits[2]` will be placed. The other variable fields `{var}` refer to the fields in the data json. \n",
+    "\n",
+    "For example, given: \n",
+    "\n",
+    "1. the data json **{\"sentence1\": \"And he said, Mama, I'm home.\", \"sentence2\": \"He didn't say a word.\"}**\n",
+    "\n",
+    "\n",
+    "2. virtual token splits set to `virtual_token_splits = [3, 3, 3]` \n",
+    "\n",
+    "\n",
+    "3. a prompt template set to `prompt_template = \"<|VIRTUAL_PROMPT_0|> Hypothesis: [sentence1], <|VIRTUAL_PROMPT_1|> Premise: [sentence2] <|VIRTUAL_PROMPT_2|> Answer:\"`\n",
+    "\n",
+    "the input will be translated into **<span style=\"color:red\">VVV</span> Hypothesis: And he said, Mama, I'm home.<span style=\"color:red\">VVV</span> Premise: He didn't say a word.<span style=\"color:red\">VVV</span> Answer:**, where <span style=\"color:red\">VVV</span> are three virtual tokens.\n",
+    "\n",
+    "Because we are only p-tuning on the Sentiment Analysis and Intent/Slot Classification tasks right now, we only need to set the task templates for those two tasks. But, we are going to go ahead and set the template for all 3 tasks here, just to show that you can set the templates for all the tasks you have planned at one time, then prompt tune/p-tune on them sequentially. \n",
+    "\n",
+    "Let's configure all of our templates for **Sentiment Analysis**, **Intent and Slot Classification**, and **Question Answering** tasks:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f935b411",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config.model.task_templates = [\n",
+    "    {\n",
+    "        \"taskname\": \"sentiment\",\n",
+    "        \"prompt_template\": \"<|VIRTUAL_PROMPT_0|> {sentence} sentiment: {label}\",\n",
+    "        \"total_virtual_tokens\": 10,\n",
+    "        \"virtual_token_splits\": [10],\n",
+    "        \"truncate_field\": None,\n",
+    "        \"answer_only_loss\": False,\n",
```

Comment:
I see you use `answer_only_loss: False` here. I only tested `True` case. Is `False` better?

Reply:
They both work fine